pytorch_lightning/core/lightning.py

```diff
@@ -1106,7 +1107,12 @@ def training_step(...):
         self.trainer.train_loop.backward(loss, optimizer, -1, *args, **kwargs)
         self._running_manual_backward = False
 
-    def manual_optimizer_step(self, optimizer: Optimizer, force_optimizer_step:bool = False) -> None:
+    def manual_optimizer_step(self,
+                              optimizer: Optimizer,
+                              *args,
+                              make_optimizer_step: Optional[bool] = None,
```

Comment:
What None for optional bool means, also not mentioned docs
```suggestion
                              make_optimizer_step: bool = False,
```

Reply:
If nothing is provided, it will use accumulated_grad_batches. I added this several times in the examples.